uniffi_bindgen/src/bindings/kotlin/templates/ErrorTemplate.kt

```diff
@@ -57,12 +55,12 @@ sealed class {{ toplevel_name }}: Exception(){% if self.contains_object_referenc
         }
     }
 
-    {% if self.contains_object_references() %}
+    {% if contains_object_references %}
     @Suppress("UNNECESSARY_SAFE_CALL") // codegen is much simpler if we unconditionally emit safe calls here
     override fun destroy() {
         when(this) {
             {%- for variant in e.variants() %}
-            is {{ e.name()|class_name_kt }}.{{ variant.name()|class_name_kt }} -> {
+            is {{ e.nm() }}.{{ e.variant_name(variant) }} -> {
```

Comment:
Avoiding the use of filters here is a massive improvement to readability of the templates.